Core/DetectorConstruction.cpp

```diff
@@ -616,6 +647,7 @@ void DetectorConstruction::ConstructTargetRun5()
   const double chamber_radius_outer_3 = 1.8 * cm;
   const double chamber_radius_outer_4 = 1.57 * cm;
   const double xadFilling_halfthickness = 0.6 * cm;
+  const double halfZ = 2.6 * cm;
```

Comment:
Maybe change the name of this variable to the same as in ConstructTargetRun3
I suggest vacuumChamber_halfLength, but you can suggest something else.